pkgs/tools/misc/hdf4/default.nix

```diff
@@ -31,11 +31,12 @@ stdenv.mkDerivation rec {
     "-DHDF4_BUILD_WITH_INSTALL_NAME=OFF"
     "-DHDF4_ENABLE_JPEG_LIB_SUPPORT=ON"
     "-DHDF4_ENABLE_NETCDF=OFF"
-    "-DHDF4_ENABLE_SZIP_ENCODING=ON"
-    "-DHDF4_ENABLE_SZIP_SUPPORT=ON"
     "-DHDF4_ENABLE_Z_LIB_SUPPORT=ON"
     "-DHDF4_BUILD_FORTRAN=OFF"
     "-DJPEG_DIR=${libjpeg}"
+  ] ++ stdenv.lib.optionals (szip != null) [
+    "-DHDF4_ENABLE_SZIP_ENCODING=ON"
```

Comment:
@7c6f434c It appears that GitHub is sloppy in the language they use when describing their features. 

`Filter pull requests that a reviewer has approved: state:open type:pr review:approved`

I have reviewed and approved https://github.com/NixOS/nixpkgs/pull/26947, but because there are new changes it's not in the `approved` state anymore, even though the UI does shows an "out of date approved" icon. 

This means, there is a race condition when referring things via search filters. I.e., the query "show me the things someone else thought were good enough at some point in time" cannot be answered by GitHub in general, unless you talk to their API and build it yourself.

So, I assumed certain things about how GitHub works based on reading documentation and unfortunately they do not implement what they claim they implement. This resulted in sending you on a wild goose chase, which as you can surely understand was not intentional. 

So, thank you for taking the time to understand what I was talking about (I wasn't very specific, because I assumed that what I was asking was trivial, which was evidently a wrong assumption).

This does provide an important lesson in that it appears that GitHub on its own is unfit for development in the enterprise. While it can provide a basis, it's not a complete product.